logic-app-using-workflow-functions/README.md

```diff
@@ -0,0 +1,29 @@
+---
+services: media-services,logic-app
+platforms: dotnetcore
+author: raffertyuy
+---
+
+# Media Services: Integrating Azure Media Services with Azure Functions & Azure Logic Apps
+
```

Comment:
Could use a lot more description details here about the Use Case scenario that is being demonstrated with this sample